src/assets/main.css

```
@@ -317,14 +316,145 @@
   .auth-footer-link {
     @apply text-red-600 hover:text-red-700 font-semibold ml-1 transition-colors;
   }
+
+  /* ============================================================
+     LOGIN – Fondo completo con tarjeta centrada (estilo luna)
+  ============================================================ */
+
+  /* Fondo con imagen de gimnasio a pantalla completa */
+  .login-bg {
+    @apply relative flex min-h-screen w-screen items-center justify-center overflow-y-auto;
+    background-image: url('/img/fondo_sin_letra.png');
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)